src/Filter/DateComp.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { DatePicker } from "antd";
+
+const DateComp = () => {
+  const { RangePicker } = DatePicker;
+
+  const onChange = ({ date, dateString }: any) => {
```

Comment:
can you use the same onChange which is present in Filter.tsx

Reply:
But it displayed undefined value in console. So i check this line.

Reply:
Do you know why its getting undefined ?

Reply:
Yes , Now i know. I changed ({ date:any, dataString:any }) so now date is displayed in my console.